core/schema.mcf

```diff
@@ -2770,6 +2770,30 @@ name: "ParkingFacility"
 typeOf: schema:Class
 subClassOf: schema:CivicStructure
 
+Node: AgriculturalFacility
+typeOf: schema:Class
+subClassOf: schema:Place
```

Comment:
Can we make these  `subCladdOf: dcs:CivicStructure`

Reply:
Made the changes. Now all four nodes are subclasses of CivicStructures